deploy/README.md

```diff
@@ -1,3 +1,13 @@
+Prerequisites
+=============
+
+Before building the Docker image, clone the SQL-to-DBSP compiler at the root of this repository and then build it:
+
+```
+git clone https://github.com/vmware/sql-to-dbsp-compiler
+(cd sql-to-dbsp-compiler/SQL-compiler/ && mvn -DskipTests package)
```

Comment:
what about required dependencies?
this is ugly, we should have a build.sh/build.bat command to do this.

Reply:
Yes, we need proper automation.  I suggest we rely on `cargo make` to automate the build process, including pulling dependencies. We should also address https://github.com/vmware/sql-to-dbsp-compiler/issues/119, so that the DBSP repo can depend on a specific compiler version. Alternatively (additionally?) we can add the sql compiler as git submodule to dbsp.